kubernetes-client/src/main/java/io/fabric8/kubernetes/client/utils/KubernetesResourceUtil.java

```diff
@@ -371,7 +360,34 @@ public static Duration getAge(HasMetadata kubernetesResource) {
       Class<?> listTypeClass = Thread.currentThread().getContextClassLoader().loadClass(type.getName() + "List");
       return (Class<KubernetesResourceList<T>>) listTypeClass;
     } catch (ClassNotFoundException | ClassCastException e) {
-      return CustomResourceList.class;
+      return (Class<? extends KubernetesResourceList>) CustomResourceList.class;
```

Comment:
why is this change required?

Reply:
That was by mistaken. Modified and rebased it with master.